lms/templates/certificates/_certificate-signatories.html

```diff
@@ -0,0 +1,14 @@
+<div>
```

Comment:
Note:  Inline CSS is acceptable in this case because a new template is in development and will be delivered in the next  sprint.